app/models/raster_resource.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class RasterResource < Valhalla::Resource
+  include Valkyrie::Resource::AccessControls
+  include Schema::Geo
+
+  attribute :id, Valkyrie::Types::ID.optional
+  attribute :member_ids, Valkyrie::Types::Array
+  attribute :member_of_collection_ids
+  attribute :state
+  attribute :workflow_note, Valkyrie::Types::Array.member(WorkflowNote).optional
+  attribute :file_metadata, Valkyrie::Types::Set.member(FileMetadata.optional)
+  attribute :pending_uploads, Valkyrie::Types::Array.member(PendingUpload)
+
+  # How is this used?
```

Comment:
I suspect this is used when a property links to a resource, and it uses the resource's `.to_s` method for display:

https://github.com/pulibrary/figgy/blob/master/app/views/catalog/_resource_attributes_default.html.erb#L23

Reply:
This was a comment carried over from Vector Works. Thanks for linking to that partial. I'll remove the comment in both places.